src/components/parser.cpp

```diff
@@ -77,7 +77,7 @@ void parser::codeblock(string&& data, const bar_settings& bar) {
 
     switch (tag) {
       case 'B':
-        m_sig.emit(change_background{parse_color(value, bar.background)});
+        m_sig.emit(change_background{parse_color(value, 0x0)});
```

Comment:
Why is this set to `0x0`?

Reply:
This is because 0 specifes "fully transparent": the bar background is now handled in the renderer by compositing everything on top of a surface filled with the background using cairo. So to use "background color" we can simply use "transparent".